pkg/types/azure/machinepool.go

```diff
@@ -36,6 +36,12 @@ type OSDisk struct {
 	DiskType string `json:"diskType"`
 }
 
+// DefaultDiskType holds the default Azure disk type used by the VMs.
+const DefaultDiskType string = "Premium_LRS"
```

Comment:
Use this constant here, too.
https://github.com/openshift/installer/blob/eb47ed1dea8b4ac393e58d1b6bd19d7edb113908/pkg/asset/machines/worker.go#L97

Reply:
Tried it and it gives me import cycle error. Any other suggestions?

Reply:
That does not make any sense. The `pkg/types/azure` package does not import any other installer packages. Can you be specific about (1) what you tried and (2) what the error was?

Reply:
Ah got it. Was trying to avoid this default variable by directly using it in the validation file. I'll make the change thanks.

Reply:
```
const DefaultDiskType string = machines.DefaultAzureMachinePoolPlatform().OSDisk.DiskType

package github.com/openshift/installer/cmd/openshift-install
	imports github.com/openshift/installer/cmd/openshift-install/migrate/azure
	imports github.com/openshift/installer/pkg/migrate/azure
	imports github.com/openshift/installer/pkg/asset/installconfig/azure
	imports github.com/openshift/installer/pkg/types
	imports github.com/openshift/installer/pkg/types/azure
	imports github.com/openshift/installer/pkg/asset/machines
	imports github.com/openshift/installer/pkg/asset/ignition/machine
	imports github.com/openshift/installer/pkg/asset/installconfig
	imports github.com/openshift/installer/pkg/asset/installconfig/azure: import cycle not allowed

```

Reply:
Yep, you've got it backwards. Keep the default defined in `pkg/asset/machines/worker.go`. Use the default in `DefaultAzureMachinePoolPlatform`.

Reply:
ah makes sense